### PR TITLE
Use camelCase for values in commutable

### PR DESCRIPTION
**Pull request by captainsafia**
Updates commutable to use camel case property names and updates tests and code to refer to them.

### CODE REVIEW

**Review by rgbkrk — CHANGES_REQUESTED** [review]
Commutable's toJS functions will need to take the `camelCase` -> `snake_case` into account since it has to match the jupyter notebook format. This also highlights that we need some conformance tests based on commutable output, having it run against the nbformat json schema.

**Comment by captainsafia** [comment]
>Commutable's toJS functions will need to take the camelCase -> snake_case into account since it has to match the jupyter notebook format. This also highlights that we need some conformance tests based on commutable output, having it run against the nbformat json schema.

Is there a particular `toJS` function you are referring? I believe I updated all the cell related ones.

File: packages/commutable/src/v4.ts [hunk 1]
```diff
@@ -285,29 +285,29 @@ const mimeBundleToJS = (immMimeBundle: ImmutableMimeBundle): MimeBundle => {
...
-  switch (output.output_type) {
+  switch (output.outputType) {
     case "execute_result":
       return {
-        output_type: output.output_type,
-        execution_count: output.execution_count,
+        output_type: output.outputType,
+        execution_count: output.executionCount,
         data: mimeBundleToJS(output.data),
         metadata: output.metadata.toJS()
```
Comment (rgbkrk) [inline packages/commutable/src/v4.ts:294]: I stand corrected, you've totally got this covered.

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
This turned out really nicely. Do we also need to update the reducers and epics that generate outputs?

**Comment by rgbkrk** [comment]
Ah crap, we do need to update display area and others before we can have this. I'm going to revert so we can have `master` in a working state.

**Comment by rgbkrk** [comment]
I'll make a new PR that uses all your commits and also making sure we adapt to the new APIs, so it'll be a one-shot PR.

**Pull request merged**